Userland/Libraries/LibJS/Runtime/DeclarativeEnvironment.h

```diff
@@ -31,6 +31,9 @@ class DeclarativeEnvironment : public Environment {
 
     void initialize_or_set_mutable_binding(Badge<ScopeNode>, GlobalObject& global_object, FlyString const& name, Value value);
 
+    // This is not a method defined in the spec! Do not use this in any LibJS (or other spec related) code.
+    [[nodiscard]] Vector<String> bindings() const;
+
```

Comment:
This feels like it could be dangerous but at the same time adding a `Badge<js>` or something didn't feel like it was much better. We can just avoid this but then we have no autocomplete on:
```
> const abc = 3;
undefined
> a<tab> -> abc
```